x-pack/plugins/ml/public/components/rule_editor/condition_expression.js

```diff
@@ -205,27 +241,13 @@ export class ConditionExpression extends Component {
             color="danger"
             onClick={() => deleteCondition(index)}
             iconType="trash"
-            aria-label="Next"
+            aria-label={this.props.intl.formatMessage({
+              id: 'xpack.ml.ruleEditor.conditionExpression.nextAriaLabel',
```

Comment:
The `aria-label` was wrong in the original code! Please could you change the id to `xpack.ml.ruleEditor.conditionExpression.deleteConditionButtonAriaLabel`